Patch from MiniMyth2 project

Developed-by: Piotr Oniszczuk <piotr.oniszczuk@gmail.com>
Signed-off-by: Piotr Oniszczuk <piotr.oniszczuk@gmail.com>




From: devmfc <devmfc@mvand.net>
Subject: [PATCH 19/32] [DEVMFC] media: cec: platform/meson: ao-cec-g12a: fix
 debugfs error message

Add name to second registered regmap to prevent debugfs error message.
---
 drivers/media/cec/platform/meson/ao-cec-g12a.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/cec/platform/meson/ao-cec-g12a.c b/drivers/media/cec/platform/meson/ao-cec-g12a.c
index 41f5b8669c..40c6f20439 100644
--- a/drivers/media/cec/platform/meson/ao-cec-g12a.c
+++ b/drivers/media/cec/platform/meson/ao-cec-g12a.c
@@ -410,6 +410,7 @@ static const struct regmap_config meson_ao_cec_g12a_cec_regmap_conf = {
 	.reg_read = meson_ao_cec_g12a_read,
 	.reg_write = meson_ao_cec_g12a_write,
 	.max_register = 0xffff,
+	.name = "cec_g12a",
 };
 
 static inline void
-- 
2.46.0

--

MiniMyth2

